components/tools/OmeroWeb/omeroweb/api/views.py

```diff
@@ -237,6 +237,40 @@ class PlateView(ObjectView):
 
     OMERO_TYPE = 'Plate'
 
+    # Urls to add to marshalled object. See ProjectsView for more details
+    urls = {
+        'url:wells': {'name': 'api_plate_wells',
+                      'kwargs': {'plate_id': 'OBJECT_ID'}}
```

Comment:
```OBJECT_ID``` is a unique identifier that this should be replaced with the ID of the current object. It's possible that other IDs could be used in future. Using ```None``` doesn't explicitly say what the placeholder should be replaced with.